src/views/userOp/UserOperation.tsx

```diff
@@ -295,427 +476,356 @@ function RecentUserOps(props: any) {
-                                                                                                    unit256
+                                                                                                <td className="whitespace-pre text-black[87%] py-[14px] px-3 text-sm leading-5">
+                                                                                                    Bytes
                                                                                                 </td>
-                                                                                                <td className="whitespace-pre text-black[87%] py-[14px] text-sm leading-5">
-                                                                                                    {getFee(item.value!, item.network)}
-                                                                                                </td>
-                                                                                                <td className="whitespace-pre text-black[87%] py-[14px] text-sm leading-5">
-                                                                                                    <span className="text-blue-200 text-sm leading-5"></span>
+                                                                                                <td className="whitespace-normal break-all text-black[87%] py-[14px] px-3 text-sm leading-5">
+                                                                                                    {item.input}
                                                                                                 </td>
                                                                                             </tr>
-                                                                                            <tr>
-                                                                                                <td className="text-black[87%] text-end text-sm leading-5 py-[14px] px-4"></td>
-                                                                                                <td className="text-black[87%] text-left text-sm leading-5 py-[14px] px-4">
+                                                                                        ) : (
+                                                                                            ''
+                                                                                        )}
+                                                                                        <tr>
+                                                                                            <td className="whitespace-pre text-black[87%] py-[14px] px-3 text-sm leading-5">
+                                                                                                <div className="flex gap-2 items-center">
                                                                                                     calldata
```

Comment:
The UserOperation.tsx file is quite large. See if you can break it down into subcomponents. 

The callData handling is one such piece of code that can be made into  component.